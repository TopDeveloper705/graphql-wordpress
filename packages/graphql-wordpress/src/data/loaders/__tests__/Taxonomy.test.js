import getTaxonomyLoaders from 'data/loaders/Taxonomy';

const Taxonomy = getTaxonomyLoaders();

describe('Test Taxonomy loader', () => {
  test('Load a taxonomy', async () => {
    const tax = await Taxonomy.load('tag');
    expect(tax.getID()).toMatchSnapshot();
    expect(tax).toMatchSnapshot();
  });
});
