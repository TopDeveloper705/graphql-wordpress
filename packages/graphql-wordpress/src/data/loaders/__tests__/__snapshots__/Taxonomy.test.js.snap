// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Taxonomy loader Load a taxonomy 1`] = `"VGF4b25vbXk6dGFn"`;

exports[`Test Taxonomy loader Load a taxonomy 2`] = `
Taxonomy {
  "slug": "tag",
}
`;
